src/internal-links/handler.js

```diff
@@ -106,7 +108,67 @@ export async function internalLinksAuditRunner(auditUrl, context, site) {
   };
 }
 
+async function convertToOpportunity(auditUrl, auditData, context) {
+  const {
+    dataAccess,
+    log,
+  } = context;
+
+  const opportunities = await dataAccess.Opportunity.allBySiteIdAndStatus(auditData.siteId, 'NEW');
+  let opportunity = opportunities.find((oppty) => oppty.getType() === AUDIT_TYPE);
+
+  if (!opportunity) {
+    const opportunityData = {
+      siteId: auditData.siteId,
+      auditId: auditData.id,
+      runbook: 'https://adobe.sharepoint.com/sites/aemsites-engineering/Shared%20Documents/3%20-%20Experience%20Success/SpaceCat/Runbooks/Experience_Success_Studio_Broken_Internal_Links_Runbook.docx?web=1',
+      type: AUDIT_TYPE,
+      origin: 'AUTOMATION',
+      title: 'Broken Internal Links',
+      description: 'Broken internal links can significantly harm website\'s SEO by preventing search engines from properly indexing site, potentially reducing search rankings',
+      guidance: {
+        steps: [
+          'Update each broken internal link to valid URLs.',
+          'Test the implemented changes manually to ensure they are working as expected.',
+          'Monitor internal links for 404 errors in RUM tool over time to ensure they are functioning correctly.',
+        ],
+      },
+      tags: [
+        'SEO',
+        'Web indexing',
+        'Web crawling',
+        'User Experience',
+        'Organic Traffic',
+      ],
+    };
+    opportunity = await dataAccess.Opportunity.create(opportunityData);
```

Comment:
add error handling as done here:
https://github.com/adobe/spacecat-audit-worker/pull/475/files#diff-212dd58956c947938959c789cc3912af8f1b884de5543486166247e12f8ef215R188

Reply:
Error handling is done by `audit.js` post processor call - so I don't think it needs to be repeated here.
https://github.com/adobe/spacecat-audit-worker/blob/462a52e48c7681667a846996e112cb6b7d0c78fa/src/common/audit.js#L139

Reply:
Adding error handling, and logging.

Reply:
added